src/tpt_reports/tpt_reports.py

```diff
@@ -0,0 +1,87 @@
+"""tpt_reports is a report generation Python library and tool.
+
+Usage:
+  tpt-reports [--log-level=LEVEL] [JSON_FILE_PATH]
+
+Options:
+  -h --help                         Show this message.
+  -l --log-level=LEVEL              If specified, then the log level will be set to
+                                    the specified value.  Valid values are "debug", "info",
+                                    "warning", "error", and "critical". [default: info]
+Arguments:
+  JSON_FILE_PATH                    Path to the JSON file to act as a data source.
+"""
+
+# Standard Python Libraries
+import json
+import logging
+
+# import os
+import sys
+from typing import Any, Dict
+
+# Third-Party Libraries
+import docopt
+from schema import And, Schema, SchemaError, Use
+
+from ._version import __version__
+
+LOGGER = logging.getLogger(__name__)
+LOGGER.addHandler(logging.StreamHandler())
+LOGGING_FILE = "phish_report_generator.log"
+
+
+# Issue #6 - Create Unit Tests
+# TODO: Add unit tests for following logic and remove this comment.
+def get_json_file(file_path):
```

Comment:
I suggest renaming this function to `load_json_file` - it's a bit more descriptive than `get_json_file`, which is somewhat vague.
```suggestion
def load_json_file(file_path):
```

Reply:
Renamed method in 751ab39f57d394b36d86a9f04d9e28c311456cbc to the suggestion.